### DISCUSSION TITLE
Self-hosted RabbitMQ as event source for Lambda?

### DISCUSSION THREAD

**Discussion by j-adamczyk** [serverless/serverless#11327] [Q&A]
From Serverless docs I know that Amazon MQ (Amazon managed RabbitMQ) is supported as event source for Lambda. I have to use ARN for that. However, can I also use self-hosted RabbitMQ? For Kafka it is possible to use self-hosted instance.

**Comment by pgrzesik** [comment]
Hey @j-adamczyk - to my best knowledge, it is currently not possible, but I believe it might change in the future. As you noted, it's possible for Kafka, but support for self-hosted Kafka was supported a bit after announcing MSK support, so we might have the same situation here.
